includes/discovery/entity-physical.inc.php

```diff
@@ -30,10 +30,16 @@
     $entPhysicalAlias             = $entry['entPhysicalAlias'];
     $entPhysicalAssetID         = $entry['entPhysicalAssetID'];
 
-    if (isset($entity_array['$entPhysicalIndex']['0']['entAliasMappingIdentifier'])) { $ifIndex = $entity_array['$entPhysicalIndex']['0']['entAliasMappingIdentifier']; }
+    if (isset($entity_array[$entPhysicalIndex]['0']['entAliasMappingIdentifier'])) {
+      $ifIndex = $entity_array[$entPhysicalIndex]['0']['entAliasMappingIdentifier'];
+    }
 
-    if (!strpos($ifIndex, "fIndex") || $ifIndex == "") { unset($ifIndex);  }
-    list(,$ifIndex) = explode(".", $ifIndex);
+    if (!strpos($ifIndex, "fIndex") || $ifIndex == "") {
+      unset($ifIndex);
+    } else {
```

Comment:
This wasn't an else before and was done after the if statement, is this correct?